From b1b9980853b1083f0c8b9f628f8b4c3a484d4f91 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Thu, 22 Jun 2017 10:25:08 +0800
Subject: [PATCH] net-snmp: fix for --disable-des

Include des.h only if it's found in openssl so that
the --disable-des works correctly.

Upstream-Status: Submitted [net-snmp-coders@lists.sourceforge.net]

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>

---
 snmplib/scapi.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/snmplib/scapi.c b/snmplib/scapi.c
index 54fdd5c..0f7e931 100644
--- a/snmplib/scapi.c
+++ b/snmplib/scapi.c
@@ -85,7 +85,9 @@ netsnmp_feature_child_of(usm_scapi, usm_support);
 #include <openssl/hmac.h>
 #include <openssl/evp.h>
 #include <openssl/rand.h>
+#ifdef HAVE_OPENSSL_DES_H
 #include <openssl/des.h>
+#endif
 #ifdef HAVE_AES
 #include <openssl/aes.h>
 #endif
